Reply/app/src/main/java/com/example/reply/ui/ReplyListContent.kt

```diff
@@ -69,20 +75,31 @@ fun ReplyInboxScreen(
         }
     }
 
+    var selectedEmailIds by remember { mutableStateOf(setOf<Long>()) }
+    val swapEmailSelection = { id: Long ->
```

Comment:
Does the swap mean user would be able to swap/drag to order the emails?

Reply:
No, sorry that was a naming issue. I renamed to "toggleEmailSelection"